core-services/prow/02_config/_boskos.yaml

```diff
@@ -1386,7 +1386,12 @@ resources:
   state: free
   type: vsphere-clusterbot-quota-slice
 - names:
-  - ci-segment-223
+  - devqe-segment-223
+  - devqe-segment-224
+  - devqe-segment-225
+  - devqe-segment-226
+  - devqe-segment-227
+  - devqe-segment-228
```

Comment:
I saw flex is using `devqe-segment-221`, are these all the left that we have?

Reply:
yes, devqe-segment-221 and devqe-segment-222 is for QE/Dev manual testing